.github/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Stock Exchange</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1 class="title">Search Nasdaq Stocks</h1>
+    </div>
+    <div class="input-btn-box" id="searchFormContainer">
```

Comment:
the form should be created from the seachForm class, instead of harcoding it with HTML tags